rosdep/base.yaml

```
@@ -460,6 +460,17 @@ castxml:
   fedora: [castxml]
   macports: [castxml]
   ubuntu: [castxml]
+catch2:
+  arch: [catch2]
+  debian: [catch2]
+  fedora:
+    '*': [catch2-devel]
+    '37': null
+  freebsd: [catch2]
+  gentoo: [catch]
+  nixos: [catch2]
+  opensuse: [Catch2]
+  ubuntu: [catch2]
```

Linter says this isn't in focal.
```suggestion
  ubuntu: [catch2]
```